test/Server.test.js

```diff
@@ -45,7 +45,7 @@ describe("Server", function() {
 		it("GET request to bundle file", function(done) {
 			request(app).get("/public/bundle.js")
 				.expect("Content-Type", "application/javascript; charset=UTF-8")
-				.expect("Content-Length", "2823")
```

Comment:
is this change specific to webpack@4?

Reply:
Yes, the `Content-Length` changed because webpack v4’s runtime code has changed between versions and now takes up a bit more space.